core/src/main/java/com/opensymphony/xwork2/ognl/SecurityMemberAccess.java

```diff
@@ -47,14 +49,35 @@
     private boolean disallowProxyMemberAccess;
 
     public SecurityMemberAccess(boolean method) {
-        super(false);
         allowStaticMethodAccess = method;
     }
 
     public boolean getAllowStaticMethodAccess() {
         return allowStaticMethodAccess;
     }
 
+    @Override
+    public Object setup(Map context, Object target, Member member, String propertyName) {
+        Object result = null;
+
+        if (isAccessible(context, target, member, propertyName)) {
+            AccessibleObject accessible = (AccessibleObject) member;
+
+            if (!accessible.isAccessible()) {
+                result = Boolean.TRUE;
```

Comment:
Will `result` be used in `restore` method as `state` parameter? if so I think it should be `Boolean.FALSE`.

Reply:
Hm... this was copy/pasted from existing code but I think you are right.